telephone-line-segments.el

```diff
@@ -30,6 +30,23 @@
 (telephone-line-defsegment telephone-line-process-segment ()
   mode-line-process)
 
+(telephone-line-defsegment* telephone-line-vc-nobackend-segment ()
+  (if vc-mode
+      (substring-no-properties vc-mode (+ 1 (string-match "[-:@!?]" vc-mode)))
+    " - "))
+
+(telephone-line-defsegment* telephone-line-buffer-shortname-segment ()
+  ;; Avoids the padding in the regular "buffer only" segment
+  (buffer-name))
```

Comment:
What's the explanation on this one again?

Reply:
This one is like the regular `(buffer-name)`, but doesn't have any padding. Like I said in the comments, maybe adding a parameter to skip padding in the regular name-only-segment is better than defining a new one.

Reply:
ah. You can just replace the existing one with this, I think it would be more expected that it works like `buffer-file-name` anyway.